tests/unit/admin/tasks/test-class-wc-payments-task-disputes.php

```diff
@@ -82,16 +81,17 @@ public function test_disputes_task_with_single_dispute_within_7days() {
 		$this->mock_cache->method( 'get_or_add' )->willReturn(
 			$mock_active_disputes
 		);
+		$disputes_task = new WC_Payments_Task_Disputes();
 
 		$this->assertEquals( 'Respond to a dispute for 20,00 €', $disputes_task->get_title() );
 		// "By <date> – <count> days left to respond"
-		$this->assertMatchesRegularExpression( '/By \w{3} \d{1,2}, \d{4} – \d+ days left to respond/', $disputes_task->get_additional_info() );
+		$this->assertMatchesRegularExpression( '/By \w+ \d{1,2}, \d{4} – \d+ days left to respond/', $disputes_task->get_additional_info() );
```

Comment:
Not suggesting a change here, but in future you could do this, self documenting regex
```php
		$this->assertMatchesRegularExpression( '/By (?P<date>\w+ \d{1,2}, \d{4}) – (?P<days>\d+) days left to respond/', $disputes_task->get_additional_info() );
```

and it has the bonus of making it easy to extract information as well.
```
array(5) {
  [0]=>
  string(41) "By Dec 7, 2023 - 3 days left to respond"
  ["date"]=>
  string(11) "Dec 7, 2023"
  [1]=>
  string(11) "Dec 7, 2023"
  ["days"]=>
  string(1) "3"
  [2]=>
  string(1) "3"
}
```